test/unit/io/structure_file/structure_file_output_test.cpp

```diff
@@ -49,68 +50,69 @@ TEST(structure_file_output_class, construct_by_filename)
     {
-        test::tmp_filename filename{"structure_file_output_constructor.xyz"};
-        EXPECT_THROW(structure_file_output<>{filename.get_path()}, unhandled_extension_error);
+        seqan3::test::tmp_filename filename{"structure_file_output_constructor.xyz"};
+        EXPECT_THROW(seqan3::structure_file_output<>{filename.get_path()}, seqan3::unhandled_extension_error);
     }
 
     /* unknown file */
     {
-        test::tmp_filename filename{"I/do/not/exist.dbn"};
-        EXPECT_THROW(structure_file_output<>{filename.get_path()}, file_open_error);
+        seqan3::test::tmp_filename filename{"I/do/not/exist.dbn"};
+        EXPECT_THROW(seqan3::structure_file_output<>{filename.get_path()}, seqan3::file_open_error);
     }
 
     /* non-existent file*/
     {
-        EXPECT_THROW(structure_file_output<>{"/dev/nonexistant/foobarOOO"}, file_open_error);
+        EXPECT_THROW(seqan3::structure_file_output<>{"/dev/nonexistant/foobarOOO"}, seqan3::file_open_error);
     }
 
     /* filename + fields */
     {
-        test::tmp_filename filename{"structure_file_output_constructor.dbn"};
-        EXPECT_NO_THROW((structure_file_output<fields<field::seq>,
-                                               type_list<format_vienna>>
-                                               {filename.get_path(), fields<field::seq>{}}));
+        seqan3::test::tmp_filename filename{"structure_file_output_constructor.dbn"};
+        EXPECT_NO_THROW((seqan3::structure_file_output<seqan3::fields<seqan3::field::seq>,
+                                                       seqan3::type_list<seqan3::format_vienna>>
+                                                       {filename.get_path(), seqan3::fields<seqan3::field::seq>{}}));
     }
 }
 
 TEST(structure_file_output_class, construct_from_stream)
 {
+    using fields_seq_id_structure = seqan3::fields<seqan3::field::seq, seqan3::field::id, seqan3::field::structure>;
+
     /* stream + format_tag */
-    EXPECT_NO_THROW((structure_file_output<fields<field::seq, field::id, field::structure>,
-                                           type_list<format_vienna>>
-                                           {std::ostringstream{}, format_vienna{}}));
+    EXPECT_NO_THROW((seqan3::structure_file_output<fields_seq_id_structure, seqan3::type_list<seqan3::format_vienna>>
+                                                   {std::ostringstream{}, seqan3::format_vienna{}}));
 
     /* stream + format_tag + fields */
-    EXPECT_NO_THROW((structure_file_output<fields<field::seq, field::id, field::structure>,
-                                           type_list<format_vienna>>
-                     {std::ostringstream{}, format_vienna{},
-                      fields<field::seq, field::id, field::structure>{}}));
+    EXPECT_NO_THROW((seqan3::structure_file_output<fields_seq_id_structure, seqan3::type_list<seqan3::format_vienna>>
+                                                   {std::ostringstream{},
```

Comment:
align the first `{` with `seqan3::structure_file_output` and not with its template args